### DISCUSSION TITLE
Problem loading templates

### DISCUSSION THREAD

**Discussion by DahlgrenDavid** [Open-Systems-Pharmacology/Forum#1505] [Questions & Problems]
I am doing the tutorial Drug-Drug Interaction Rifampicin/Midazolam 
(https://github.com/Open-Systems-Pharmacology/PharMetrX-Workshop-April-2021/tree/main/optional%20Hands-On%20%20-%20DDI%20Rifampicin)

In the first step I am asked to load the individual "standard European male for DDI" from the individual templates database. However, when I try to do that I get a notification "no template 'individual' available in the template databases".

Are there no individual templates available, or am I ding something else wrong?

Best
David

**Comment by tobiasK2001** [comment]
Dear David,

I assume you are working with PK-Sim version 11. I think the Workshop material refers to Version 9. Somehow the workflow to load an individual from the template database has changed in Version 11. But you can simply create a new individual from scratch using default values to go on.

Hope that might help.

Best, Tobias

**Reply by DahlgrenDavid** [reply]
Thank you Tobias. I will try that!
Best
David
